python/tmux_env.py

```diff
@@ -84,18 +92,31 @@ def config_cb(data, option, value):
         set_timer()
     return w.WEECHAT_RC_OK
 
+
 def timer_cb(buffer, args):
     """Check if tmux is attached, update environment"""
 
-    attached = os.access(SOCK, os.X_OK) # X bit indicates attached
+    attached = os.access(SOCK, os.X_OK)  # X bit indicates attached
     if attached:
         update_environment()
     return w.WEECHAT_RC_OK
 
+
+def do_check_output(command_args):
```

Comment:
@flashcode If you know of some other way to support methods with different function signatures between Python 2 and 3, I'd be happy to replace this with something better. :)